Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -1006,7 +1006,7 @@ class Gx2Fitter {
     if (trackContainer.hasColumn(
             Acts::hashString(Gx2fConstants::gx2fnUpdateColumn))) {
       ACTS_DEBUG("Add nUpdate to track")
-      track.template component<std::size_t>("Gx2fnUpdateColumn") = nUpdate;
+      track.template component<std::uint32_t>("Gx2fnUpdateColumn") = nUpdate;
```

Comment:
SonarQube points out that this is an implicit conversion, which could potentially have made explicit here.